package/Dockerfile.agent

```diff
@@ -1,4 +1,6 @@
 FROM alpine:3.12.3
+ARG ARCH
+ENV ARCH=$ARCH
 RUN apk add -U --no-cache git bash
-COPY bin/fleetagent bin/fleet package/log.sh /usr/bin/
+COPY bin/fleetagent-linux-$ARCH bin/fleet-linux-$ARCH package/log.sh /usr/bin/
```

Comment:
same here. Make sure the name is fleet-agent and fleet otherwise the code will not find binaries.

Reply:
Same as above. This _should_ have been the case. Good catch.